src/tracing.rs

```diff
@@ -57,14 +58,36 @@ impl Binding for TraceLevel {
     }
 }
 
+impl TraceLevel {
+    /// Attempt to convert this [TraceLevel] to a [log::LevelFilter].
+    ///
+    /// This function is not public to avoid having a public dependency on [`log`].
+    ///
+    /// This is done trivially with two exceptions:
+    /// - [TraceLevel::None] goes to [None]
+    /// - [TraceLevel::Fatal] goes to [log::Level::Error].
+    const fn as_log_level(self) -> Option<log::Level> {
+        use log::Level;
+
+        match self {
+            Self::None => None,
+            Self::Fatal | Self::Error => Some(Level::Error),
```

Comment:
This has the same symptom. Different users may have different thoughts on mapping Fatal and Error to different levels. We try not to decide this kind of application logic for users.